libraries/wallet/wallet.cpp

```diff
@@ -425,32 +426,29 @@ namespace golos { namespace wallet {
                     std::string author,
                     std::string title,
                     std::string memo,
-                    time_point_sec expiration,
-                    time_point_sec review_period_time,
+                    std::string expiration,
+                    std::string review_period_time,
                     bool broadcast
                 ) {
                     FC_ASSERT(_builder_transactions.count(handle));
                     proposal_create_operation op;
                     op.author = author;
                     op.title = title;
                     op.memo = memo;
-                    if (expiration.sec_since_epoch() == 0) { 
-                        auto dyn_props = _remote_database_api->get_dynamic_global_properties();
-                        op.expiration_time = dyn_props.time;
-                        op.expiration_time += STEEMIT_MAX_PROPOSAL_LIFETIME_SEC;
-                    } else {
-                        op.expiration_time = expiration;
-                    }
+
+                    auto dyn_props = _remote_database_api->get_dynamic_global_properties();
+
+                    op.expiration_time = time_converter(expiration,
+                        dyn_props.time, dyn_props.time + STEEMIT_MAX_PROPOSAL_LIFETIME_SEC).time();
 
                     // copy tx to avoid malforming if sign_transaction fails
                     signed_transaction trx = _builder_transactions[handle];
                     std::transform(
                         trx.operations.begin(), trx.operations.end(), std::back_inserter(op.proposed_operations),
                         [](const operation& op) -> operation_wrapper { return op; });
 
-                    if (review_period_time > time_point_sec::min()) {
-                        op.review_period_time = review_period_time;
-                    }
+                    op.review_period_time = time_converter(review_period_time,
```

Comment:
An ability to create proposal without review period should exist.
Add checking of `review_period_time`  on empty. 
And set `op.review_period_time` in case if `review_period_time` is not empty.